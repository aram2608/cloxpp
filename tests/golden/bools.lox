print true;            // expect true
print false;           // expect false
print !true;           // expect false
print !false;          // expect true