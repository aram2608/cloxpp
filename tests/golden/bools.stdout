Token: PRINT print 1
Token: TRUE true 1
Token: SEMICOLON ; 1
Token: PRINT print 2
Token: FALSE false 2
Token: SEMICOLON ; 2
Token: PRINT print 3
Token: BANG ! 3
Token: TRUE true 3
Token: SEMICOLON ; 3
Token: PRINT print 4
Token: BANG ! 4
Token: FALSE false 4
Token: SEMICOLON ; 4
Token: EOF  4
